__tests__/app.test.js

```diff
@@ -82,3 +83,43 @@ describe('GET/api/articles', () => {
       });
   });
 });
+
+describe.only('GET/api/articles/:article_id', () => {
+  test('status 200, responds with article object', () => {
+    return request(app)
+      .get('/api/articles/3')
+      .expect(200)
+      .then(({ body }) => {
+        const { article } = body;
+        expect(article[0]).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
If you've request article 3, you need to check you're getting article 3

at the moment, this would pass even if you got the wrong one